scripts/nolints.txt

```diff
@@ -1999,25 +1999,7 @@ apply_nolint pell.xz doc_blame
 apply_nolint pell.yz doc_blame
 
 -- order/basic.lean
-apply_nolint classical.DLO doc_blame
```

Comment:
No need to commit changes to this file. The bot will take care of updating it.

Reply:
I wanted to make sure that I added all `ge_or_gt` nolints.

Reply:
Oh, I usually just put a `#lint` at the bottom of the file and track the remaining linting errors there.